docs/src/man/data.md

```diff
@@ -0,0 +1,41 @@
+# Saving and Viewing PowerSystems Data
```

Comment:
I'm open to any changes (or deletion).  I find this kind of filtering really useful.  These commands can be difficult to remember, so I wanted to have them documented in a convenient place.